src/vscode-bicep/src/commands/generateParams.ts

```diff
@@ -34,6 +35,15 @@ export class GenerateParamsCommand implements Command {
         }
       );
       this.outputChannelManager.appendToOutputChannel(generateParamsOutput);
+
+      const filePath = path.parse(documentUri.fsPath);
+
+      const openPath = vscode.Uri.parse(
```

Comment:
The best way to handle this is to have the generateParams command return the path that it created/updated. Then there's one source of truth.

If you don't choose to go that route, you need to be careful about parsing URIs in vscode/typescript.

Use instead this (approproximately):
```
const openPath = Uri.file(path.join(filePath.dir, `${filePath.name}.parameters.json`));
```

Reply:
I'd like to go with the latter approach, since I don't have enough knowledge to change the return value of the `workspace/executeCommand` function. I may not even understand how this code works, yet.

Created commit to update the approach with the suggestion, thanks..